tools/test-backend

```diff
@@ -204,72 +217,118 @@ def main() -> None:
-    parser.add_argument('args', nargs='*')
+    parser = argparse.ArgumentParser(
+        description=usage, formatter_class=argparse.RawTextHelpFormatter,
+    )
+
+    parser.add_argument(
+        "--nonfatal-errors",
+        action="store_false",
+        default=True,
+        dest="fatal_errors",
+        help="Continue past test failures to run all tests",
+    )
+    parser.add_argument(
+        "--coverage",
+        dest="coverage",
+        action="store_true",
+        default=False,
+        help="Compute test coverage.",
+    )
+    parser.add_argument(
+        "--verbose-coverage",
+        dest="verbose_coverage",
+        action="store_true",
+        default=False,
+        help="Enable verbose print of coverage report.",
+    )
+    parser.add_argument(
+        "--no-cov-cleanup",
+        dest="no_cov_cleanup",
+        action="store_true",
+        default=False,
+        help="Do not clean generated coverage files.",
+    )
+
+    parser.add_argument(
+        "--parallel",
+        dest="processes",
+        type=int,
+        action="store",
+        default=default_parallel,
+        help="Specify the number of processes to run the "
+        "tests in. Default is the number of logical CPUs",
+    )
+    parser.add_argument(
+        "--profile",
+        dest="profile",
+        action="store_true",
+        default=False,
+        help="Profile test runtime.",
+    )
+    parser.add_argument(
+        "--force",
+        dest="force",
+        action="store_true",
+        default=False,
+        help="Run tests despite possible problems.",
+    )
+    parser.add_argument(
+        "--no-shallow",
+        dest="no_shallow",
+        action="store_true",
+        default=False,
+        help="Don't allow shallow testing of templates (deprecated)",
+    )
+    parser.add_argument(
+        "--verbose",
+        dest="verbose",
+        action="store_true",
+        default=False,
+        help="Show detailed output",
+    )
+    parser.add_argument(
+        "--reverse",
+        dest="reverse",
+        action="store_true",
+        default=False,
+        help="Run tests in reverse order.",
+    )
+    parser.add_argument(
+        "--rerun",
+        dest="rerun",
+        action="store_true",
+        default=False,
+        help=(
+            "Run the tests which failed the last time "
+            "test-backend was run.  Implies --nonfatal-errors."
+        ),
+    )
+    parser.add_argument(
+        "--include-webhooks",
+        dest="include_webhooks",
+        action="store_true",
+        default=False,
+        help=("Include webhook tests.  By default, they are skipped for performance."),
+    )
+    parser.add_argument(
+        "--generate-stripe-fixtures",
+        dest="generate_stripe_fixtures",
+        action="store_true",
+        default=False,
+        help=("Generate Stripe test fixtures by making requests to Stripe test network"),
+    )
+    parser.add_argument("args", nargs="*")
```

Comment:
Whee turning 50 lines of argparse into 100.  But I think we're being a bit weird with `default` and `dest` here being the default values and cleaning that up will make this happy.